src/platforms/common/shutdown.cpp

```diff
@@ -191,7 +191,11 @@ void shutdown_worker(void *arg)
 
 		} else {
 			PX4_WARN("Shutdown NOW. Good Bye.");
+#if defined __PX4_POSIX
```

Comment:
I'd rather implement board_shutdown for SITL, then we don't need this ifdef.
`px4_systemreset` is incorrect here, as it's supposed to do a reboot.